lib/pliny/sidekiq/middleware/server/log.rb

```diff
@@ -1,8 +1,7 @@
 module Pliny::Sidekiq::Middleware
   module Server
     class Log
-      def initialize(metric_prefix: nil)
-        @metric_prefix = metric_prefix
+      def initialize(opts={})
```

Comment:
No, it's here for compatibility with Sidekiq's middleware. I actually tried removing this, but it went :boom:.